hack/test-cmd.sh

```diff
@@ -1,143 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
keep this file so you don't break our current CI.  You can change it's content so that you have a clean spot to start the new tests after the clsuter comes up.